cooperator/models/subscription_request.py

```diff
@@ -143,12 +151,17 @@ def check_iban(self, iban):
             return False
 
     @api.multi
-    @api.depends("firstname", "lastname")
+    @api.depends("firstname", "lastname", "company_name")
     def _compute_name(self):
         for sub_request in self:
-            sub_request.name = " ".join(
-                part for part in (sub_request.firstname, sub_request.lastname) if part
-            )
+            if sub_request.is_company:
+                sub_request.name = self.company_name
```

Comment:
Cela ne va-t-il pas écraser le nom du représentant ?

Reply:
yes, but the name of a subscription request for a company is supposed to be the name of the company (as it was in 9 and in 12 before the changes to the `name` field). this allows to search by the company name. `firstname` and `lastname` are still set to the name of the representative.

Reply:
OK :+1: